Update broadcast.md (#695)
xmtp/xmtp-dot-org: docs/tutorials/broadcast.md

You can send a broadcast message (1:many message or announcement) with XMTP. The recipient sees the message as a DM from the sending wallet address.

1. Use the bulk query `canMessage` method to identify the wallet addresses that are activated on the XMTP network, up to 1k per batch.
2. Begin sending messages to the activated wallet addresses.
3. Your sending logic should monitor responses from the XMTP network and respond as follows:

>    - If you recieve a 200 response, send the next message.

Reviewer: ```suggestion
   - If you receive a 200 response, send the next message.
```

> - If you recieve a 200 response, send the next message.
> - If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.

Reviewer 2: ```suggestion
  - If you recieve a 200 response, send the next message.
  - If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.
```

>     If you recieve a 200 response, send the next message.
>     If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.

Reviewer 2: ```suggestion
- If you recieve a 200 response, send the next message.
- If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.
```

>   - If you recieve a 200 response, send the next message.
>   - If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.

Reviewer 2: ```suggestion
   - If you recieve a 200 response, send the next message.
   - If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.
```

>    - If you recieve a 429 response, pause sending for one minute then resume sending starting with the message which triggered the 429.

Reviewer: ```suggestion
   - If you receive a 429 response, pause sending for one minute then resume sending starting with the message that triggered the 429.
```

Reviewer: Do I need to wait one minute - or five minutes?

Author: Per Nick: "The rate limiting rules are a little fuzzy. To avoid excessive waiting they should try again every minute, and if the first message fails it hits the brakes for another minute. That way if they only needed to wait 3 minutes they don't waste 2 minutes doing nothing."

> 4. Repeat the loop described in step 3 until all activated wallet addresses have been messaged.

Reviewer: ```suggestion
4. Repeat the loop described in step 3 until all the activated wallet addresses have been messaged.
```
